src/bfx_tools/star.ml

```diff
@@ -131,12 +131,12 @@ let align
   let reference_dir = (Filename.dirname reference_fasta#product#path) in
   let star_index_dir = sprintf "%s/star-index/" reference_dir in
   (* STAR appends Aligned.sortedByCoord.out.bam to the filename *)
-  let result = sprintf "%sAligned.sortedByCoord.out.bam" result_prefix in
+  let result = sprintf "%sAligned.out.bam" result_prefix in
```

Comment:
I guess the comment should be now saying that this is the observed filename with `--outSAMtype BAM Unsorted`